backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -1,92 +1,111 @@
+import static gov.cdc.usds.simplereport.api.Translators.parseString;
+import static gov.cdc.usds.simplereport.api.Translators.parseUserShortDate;
+import static gov.cdc.usds.simplereport.api.Translators.parseUUID;
+import static gov.cdc.usds.simplereport.api.Translators.parseYesNo;
+
 import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.RuntimeJsonMappingException;
 import com.fasterxml.jackson.dataformat.csv.CsvMapper;
 import com.fasterxml.jackson.dataformat.csv.CsvParser;
 import com.fasterxml.jackson.dataformat.csv.CsvSchema;
-import com.google.i18n.phonenumbers.NumberParseException;
-import com.google.i18n.phonenumbers.PhoneNumberUtil;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import gov.cdc.usds.simplereport.config.AuthorizationConfiguration;
+import gov.cdc.usds.simplereport.api.model.errors.IllegalGraphqlArgumentException;
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.time.format.DateTimeFormatter;
-import java.time.LocalDate;
 import java.util.Map;
-import java.util.UUID;
 
 /**
  * Created by nickrobison on 11/21/20
  */
 @Service
 @Transactional
 public class UploadService {
-    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("M/d/yyyy");
-
     private static final CsvSchema PERSON_SCHEMA = personSchema();
     private static final String FACILITY_ID = "facilityId";
 
     private final PersonService _ps;
 
-	private Map<String, String> ethnicityMap = Map.of(
-		"Hispanic or Latino", "hispanic",
-		"Not Hispanic", "not_hispanic"
-	);
-
-	private Map<String, Boolean> yesNoMap = Map.of(
-		"Yes", true,
-		"No", false
-	);
-
     public UploadService(PersonService ps) {
         this._ps = ps;
     }
 
-    @AuthorizationConfiguration.RequirePermissionExportTestEvent
-    public void processPersonCSV(InputStream csvStream) throws IOException {
-        final MappingIterator<Map<String, String>> valueIterator = new CsvMapper()
+    private MappingIterator<Map<String, String>> getIteratorForCsv(InputStream csvStream) throws IllegalGraphqlArgumentException {
+        try {
+            return new CsvMapper()
                 .enable(CsvParser.Feature.FAIL_ON_MISSING_COLUMNS)
                 .readerFor(Map.class)
                 .with(PERSON_SCHEMA)
                 .readValues(csvStream);
+        } catch (IOException e) {
+            throw new IllegalGraphqlArgumentException(e.getMessage());
+        }
+    }
+
+    private Map<String, String> getNextRow(MappingIterator<Map<String, String>> valueIterator) throws IllegalGraphqlArgumentException {
+        try {
+            return valueIterator.next();
+        } catch (RuntimeJsonMappingException e) {
+            throw new IllegalGraphqlArgumentException(e.getMessage());
+        }
+    }
+
+    @AuthorizationConfiguration.RequireGlobalAdminUser
+    @Transactional
```

Comment:
Repeated annotation is not needed unless you're changing an attribute (e.g. `readOnly=false`)

Reply:
I was scared not to have it. But now that there is a test to ensure there aren't duplicate uploads. this seems fine to remove